src/map/pc.cpp

```diff
@@ -11751,7 +11749,8 @@ static void pc_unequipitem_sub(struct map_session_data *sd, int n, int flag) {
 	}
 
 	if (flag & 1 || status_calc) {
-		pc_checkallowskill(sd);
+		if (!(flag & 4))
+			pc_checkallowskill(sd);
```

Comment:
```suggestion
		pc_checkallowskill(sd);
```
Shouldnt that still be outside of the if?
Otherwise we lose checks that dont use flag 4 and the combo is broken?

Reply:
So `pc_checkallowskill()` only looks for required weapon or shield items. I don't think combos or anything else should affect this check specifically. Unless some combo is auto-unequipped when one of them is off?